tensorboard/plugins/hparams/backend_context.py

```diff
@@ -311,6 +311,10 @@ def _compute_hparam_info_from_values(self, name, values):
         ):
             result.domain_discrete.extend(distinct_values)
 
+        # If the result is a boolean, domain should be True and False
```

Comment:
This is probably a low value comment since the next two lines of code very clearly say the same thing.